components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlinx/coroutines/Utils.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.kotlinx.coroutines
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+/**
+ *
+ * Returns a function that limits the executions of the [block] function, until the [skipTimeInMs]
+ * passes, then the latest value passed to [block] will be used. Any calls before [skipTimeInMs]
+ * passes will be ignored. It's spected that calls to the returned function
```

Comment:
nit: "expected"? I would write: All calls to the returned function must happen on the same thread.

Reply:
👍